src/collections/Form/Form.js

```diff
@@ -63,7 +63,7 @@ export default class Form extends Component {
   }
 
   componentWillUnmount() {
-    this.element.off()
+    _.invoke(this, 'element.off')
```

Comment:
These don't actually matter since we moving away from jQuery, though, we simply calling this if it exists now.  In tests, there is no `off()`.